src/avaliador-sintatico/index.ts

```diff
@@ -115,7 +115,11 @@ export class Parser implements AvaliadorSintaticoInterface {
   }
 
   estaNoFinal(): boolean {
-    return this.peek().tipo === tiposDeSimbolos.EOF;
+    if(this.peek() && (this.peek().tipo === tiposDeSimbolos.PONTO_E_VIRGULA)){
```

Comment:
Eu atribuiria `this.peek()` pra uma variável.

Reply:
8252d3c